packages/manager/apps/web/client/app/app.module.js

```diff
@@ -285,13 +285,15 @@ export default (containerEl, environment) => {
         set(OvhHttpProvider, 'returnErrorKey', 'data'); // By default, request return error.data
       },
     ])
+
     .config([
       '$compileProvider',
       '$logProvider',
       'constants',
       ($compileProvider, $logProvider, constants) => {
         // Debug mode and logs are disabled in production
         $compileProvider.debugInfoEnabled(!constants.prodMode);
+        $compileProvider.aHrefSanitizationWhitelist(SANITIZATION.regex);
```

Comment:
Seems that `aHrefSanitizationWhiteList` is deprecated, and should be replaced by `aHrefSanitizationTrustedUrlList` (See: https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationWhitelist)

Reply:
Hello, if I'm not mistaken, we use 1.7.9 version of angularJs, aHrefSanitizationTrustedUrlList does not exists : https://code.angularjs.org/1.7.9/docs/api/ng/provider/$compileProvider

Reply:
Yes good point, so ignore my comment then ;)